test/component/login_test.dart

```diff
@@ -47,7 +48,16 @@ void main() {
     await tester.tap(loginButton);
     await tester.pumpAndSettle();
 
-    // Check that pressing login redirects to the homepage
+    // Check that pressing login redirects to the create account page
+    final createAccountPage = find.byType(CreateAccountPage);
+    expect(createAccountPage, findsOneWidget);
+
```

Comment:
Nice job ! I just have a little suggestion to check that the fields are correctly displayed.

```suggestion

    // Check that the unique username field is displayed
    final uniqueUsernameField =
        find.byKey(CreateAccountPage.uniqueUsernameFieldKey);
    expect(uniqueUsernameField, findsOneWidget);

    // Check that the pseudo field is displayed
    final pseudoField = find.byKey(CreateAccountPage.pseudoFieldKey);
    expect(pseudoField, findsOneWidget);
```

Reply:
That's a good idea! 
I just thought that implementing them directly in `login_test` was a bit weird, so I created a new test file. Applied in e8e876789c7e5ea87f7bc1e963ef064bf128c55a.